README.md

```diff
@@ -2,31 +2,31 @@
 
 
 ## Overview
-OpenFin FDC3 provides an implementation of the [FINOS FDC3](https://fdc3.finos.org/) standards for OpenFin-based applications.
+
+The OpenFin FDC3 Service provides an implementation of the [FINOS FDC3](https://fdc3.finos.org/) standards for OpenFin-based applications.
 
 This project consist of 3 parts:
-1. The FDC3 Service, taking care of intents, context and resolving them (UI)
-2. The FDC3 Client, exposing APIs for applications to handle/raise intents with contexts
+1. The FDC3 Provider, taking care of intents, context and resolving them (UI)
```

Comment:
Maybe switch context & intents around as you resolve intents, not contexts?

Reply:
✅